docs/gettingstarted/quickstart_guide.md

```diff
@@ -4,6 +4,8 @@
 - Create the following default indexes that are used by SC4S
     * email
     * epav
+    * fireeye
```

Comment:
[using here](https://github.com/splunk/splunk-connect-for-syslog/blob/3109c3ebc8a580d696584de0c01725d50f8ac9b5/package/etc/conf.d/conflib/cef/app-cef-fireeye_etp.conf#L5)